src/utils/blocks.ts

```diff
@@ -1,6 +1,43 @@
 import {Block, CustomConfig, PCShareSocialNetwork, TextSize} from '../models';
 import {ConstructorBlock} from '../models/constructor';
 
+const BLOCK_ELEMENTS = [
```

Comment:
With which element we could render span in HTML?

Reply:
But why you chose all these to render `div`?

Reply:
In HTML component we can choose div or span tag. And in this case I check if child component is block element, if it is I take div as parent. Otherwise we take default 'span' tag.